bin/index.js

```diff
@@ -106,6 +106,29 @@ const buildConfigurationDescriptor = (manifest) => {
   };
 };
 
+const addExchangeUrl = (manifest) => {
+  return inquirer.prompt([
+    {
+      type: 'input',
+      name: 'exchangeUrl',
+      message: 'Please provide the URL to your extension’s listing on Adobe Exchange. ' +
+        'Leave it blank if you don\'t have one.',
+      validate(input) {
+        if (input.length && !input.match(/^https:\/\/www\.adobeexchange\.com\/experiencecloud\.details\..+\.html$/ig)) {
```

Comment:
Inside the developer documentation, we are saying: `It must match the pattern https://www.adobeexchange.com/experiencecloud.details.######.html.`.

Inside the turbine schemas, we are validation using the RegExp: `^https:\\/\\/(?:(?:experiencecloud|www)\\.)?adobeexchange\\.com`. Probably we should update the RegExp inside turbine schemas.